src/providers/FileSystemProvider/FileSearchProvider.ts

```diff
@@ -19,9 +19,12 @@ export class FileSearchProvider implements vscode.FileSearchProvider {
     let counter = 0;
     let pattern = query.pattern.charAt(0) == "/" ? query.pattern.slice(1) : query.pattern;
 
-    // Drop a leading **/ from the glob pattern if it exists (added by Find widget of Explorer tree, which since 1.94 uses FileSearchProvider)
+    // Drop a leading **/ from the glob pattern if it exists. This gets added by Find widget of Explorer tree (non-fuzzy mode), which since 1.94 uses FileSearchProvider
     if (pattern.startsWith("**/")) {
       pattern = pattern.slice(3);
+    } else if (pattern.length && !pattern.includes("*") && options.useGlobalIgnoreFiles) {
```

Comment:
I don't think this check is necessary. I think every search using this provider is supposed to be fuzzy. This is the documentation for `query.pattern`:

> The search pattern to match against file paths. To be correctly interpreted by Quick Open, this is interpreted in a relaxed way. The picker will apply its own highlighting and scoring on the results.
> 
> Tips for matching in Quick Open: With the pattern, the picker will use the file name and file paths to score each entry. The score will determine the ordering and filtering. The scoring prioritizes prefix and substring matching. Then, it checks and it checks whether the pattern's letters appear in the same order as in the target (file name and path). If a file does not match at all using our criteria, it will be omitted from Quick Open.